features/comments_and_kudos/kudos.feature

```diff
@@ -51,6 +51,10 @@ Feature: Leave kudos
     When I am logged in as "myname1"
       And I view the work "Awesome Story"
       Then I should not see "Kudos ♥"
+    When I am logged in as "myname2"
```

Comment:
If we reopen this file, it would be a good idea to add a comment at line 51 to explain that this test is to check you can't kudos your own work. Even better, this kudos test could be split into several smaller ones, but I'll pick that up another time - not urgent or important for this PR.